carmin-server/server/test/fakedata/users.py

```diff
@@ -16,3 +16,11 @@ def standard_user(encrypted=False):
         password=generate_password_hash("user") if encrypted else "user",
         role=Role.user,
         api_key="standard-user-api-key")
+
+
+def standard_user_2(encrypted=False):
```

Comment:
What is the use of having unencrypted passwords written to the database for testing?